audit-ci.json

```diff
@@ -6,14 +6,14 @@
       "GHSA-gpv5-7x3g-ghjv": {
         "active": true,
         "notes": "no fix available as of 20230622",
-        "expiry": "2023-06-26"
+        "expiry": "2023-07-07"
       }
     },
     {
       "GHSA-c2qf-rxjj-qqgw": {
         "active": true,
         "notes": "no fix available as of 20230622",
-        "expiry": "2023-06-26"
```

Comment:
Fixes **are** available, these allowlists do not need to be extended

Reply:
removed.